packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -96,11 +145,25 @@ export const Control = ({
     }
   }, [isExpanded, setIsExpanded]);
```

Comment:
```suggestion
const toggleMeasures = useCallback(() => {
    setIsExpanded(expanded => !expanded);
  }, [isExpanded, setIsExpanded]);
```
For toggles, I like to use the callback function to make it more concise and it's the most reliable way to do the toggle.